src/Interceptor/Hsts/GooglePreloadListJar.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Amp\Http\Client\Interceptor\Hsts;
+
+final class GooglePreloadListJar extends ReadOnlyHstsJar
```

Comment:
Reads from the Google preload list https://hstspreload.org/ - not an official standard but the de facto preload list for most browsers (Chrome, Firefox, Opera, Safari, IE 11+ and Edge)